source/config/Config.cpp

```diff
@@ -121,6 +121,10 @@ bool PlainConfig::LoadFromJson(const Crt::JsonView &json)
             auto path = FileUtils::ExtractExpandedPath(json.GetString(jsonKey).c_str());
             if (FileUtils::FileExists(path))
             {
+                FileUtils::ValidateFilePermissions(
+                    FileUtils::ExtractParentDirectory(path), Permissions::ROOT_CA_DIR, false);
+                FileUtils::ValidateFilePermissions(path, Permissions::ROOT_CA, false);
```

Comment:
This should fail with fatal error. It is a security risk to allow files with incorrect file permissions.